src/seedu/addressbook/ui/TextUi.java

```diff
@@ -12,29 +11,124 @@
 import java.util.Optional;
 import java.util.Scanner;
 
-/**
- * Text UI of the application.
- */
-public class TextUi {
 
-    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+class Formatter {
```

Comment:
could be better if you put Formatter into a separate file and add header comments for the class